tests/multi_objective_tests/samplers_tests/test_samplers.py

```diff
@@ -81,6 +81,21 @@ def test_random_mo_sampler_reseed_rng() -> None:
     assert str(original_random_state) != str(sampler._sampler._rng.get_state())
 
 
+@pytest.mark.parametrize(
+    "sampler_class",
+    [
+        optuna.multi_objective.samplers.RandomMultiObjectiveSampler,
+        optuna.multi_objective.samplers.NSGAIIMultiObjectiveSampler,
+        optuna.multi_objective.samplers.MOTPEMultiObjectiveSampler,
+    ],
+)
```

Comment:
Alternatively, we might add `MOTPEMultiObjectiveSampler` to https://github.com/optuna/optuna/blob/af2ba57dc034660fe77979959a5313b08c62bd74/tests/multi_objective_tests/samplers_tests/test_samplers.py#L16-L22.

Reply:
Not a strong opinion, but I think we do not need to improve the test coverage of the deprecated sampler. So, I prefer the current change.